sdk/cosmos/azure-cosmos-spark_3_2-12/docs/configuration-reference.md

```diff
@@ -73,7 +73,7 @@ Used to influence the json serialization/deserialization behavior
 | `spark.cosmos.partitioning.targetedCount`      | None    | The targeted Partition Count. This parameter is optional and ignored unless strategy==Custom is used. In this case the Spark Connector won't dynamically calculate number of partitions but stick with this value.  |
 | `spark.cosmos.partitioning.feedRangeFilter`      | None    | Can be used to scope the query to a single logical Cosmos partition (or a subset of logical partitions). If this parameter is optionally provided, the partitioning strategy will be modified - only partitions for the scoped logical partitions will be created. So, the main benefit of this config option is to reduce the necessary SparkTasks/Partitions. |
 
-### Throughput Control Config
+#### Throughput Control Config
```

Comment:
headers seem broken in the entire file - throughput control should not be sub-section of read/write - but its own section, patch config should be in write section

Reply:
updated. 
moved patch config to be sub-section of Write config
changed Throughput control config back to its own section